Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Verification/AddressCheck.php

```diff
@@ -74,7 +74,14 @@ public function check(Payone_Api_Request_AddressCheck $request)
             $response = $this->getMapperResponse()->map($responseRaw);
 
             $this->protocol($request, $response);
-        } catch (Exception $e) {
+
+            // throw exception for testing of connection failures
```

Comment:
shall i remove this completely or should i add a check that this function is only available for Mode= Testing?